pkg/image/validation/os.go

```diff
@@ -181,6 +182,22 @@ func validateSuma(os *image.OperatingSystem) []FailedValidation {
 func validatePackages(os *image.OperatingSystem) []FailedValidation {
 	var failures []FailedValidation
 
+	if len(os.Packages.PKGList) == 0 {
+		if len(os.Packages.AdditionalRepos) > 0 {
+			failures = append(failures, FailedValidation{
+				UserMessage: "The 'packageList' field is required if any entries are specified under 'additionalRepos'.",
```

Comment:
Isn't this conflicting with the use-case where we have a RPM (in the `rpms` dir) that has some dependencies that should be retrieved from a third-party repository?

Then we would have a configuration where `AdditionalRepos` is defined without an entry in the package list.

Reply:
Thanks, I didn't catch that at all. Added a comment to explain it for other developers.